test_community/models/test_vblls.py

```diff
@@ -353,18 +353,29 @@ def test_shape_of_predictions(self) -> None:
         model = VBLLModel(
             in_features=d, hidden_features=4, out_features=1, num_layers=1
         )
-        X, y = _reg_data_singletask(d)
+        X_train, y_train = _reg_data_singletask(d)
         optim_settings = _get_fast_training_settings()
+        model.fit(X_train, y_train, optimization_settings=optim_settings)
+
+        valid_inputs = [
+            torch.rand(3, d, dtype=torch.float64),  # (3, d)
+            torch.rand(1, 3, d, dtype=torch.float64),  # (1, 3, d)
+            torch.rand(2, 3, d, dtype=torch.float64),  # (2, 3, d)
+            torch.rand(1, d, dtype=torch.float64),  # (1, d)
+        ]
+
+        for X_test in valid_inputs:
+            X_test = X_test.to(dtype=torch.float64)
```

Comment:
this seems to be a noop given that you define the tensors with this dtype above?